proto/rill/admin/v1/api.proto

```diff
@@ -1127,6 +1127,14 @@ message GetGithubUserStatusResponse {
   string access_token = 3;
   string account = 4;
   repeated string organizations = 5;
+  bool is_user_app_installed = 6;
+  string user_installation_permission =7;
```

Comment:
Could `is_user_app_installed` be removed? Seems like `user_installation_permissions` already indicates it (i.e. by not having any permissions if not installed). Or is there a difference?

Reply:
yeah it can be removed, wanted to keep it separate to keep things clean but can be removed and if `user_installation_permissions` is empty string then app is not installed on user github account ?